src/Umbraco.Core/Webhooks/WebhookEventCollectionBuilder.cs

```diff
@@ -78,4 +71,28 @@ private void RegisterTypes(IServiceCollection services)
 
         return null;
     }
+
+    public WebhookEventCollectionBuilder AddCoreWebhooks()
+    {
+        Append<ContentDeleteWebhookEvent>();
+        Append<ContentPublishWebhookEvent>();
+        Append<ContentUnpublishWebhookEvent>();
+        Append<MediaDeleteWebhookEvent>();
+        Append<MediaSaveWebhookEvent>();
+        return this;
+    }
+
+    public WebhookEventCollectionBuilder AddDictionaryWebhooks()
+    {
+        Append<DictionaryItemDeletedWebhookEvent>();
+        Append<DictionaryItemSavedWebhookEvent>();
+        return this;
+    }
+
+    public WebhookEventCollectionBuilder AddLanguageWebhooks()
+    {
+        Append<LanguageDeletedWebhookEvent>();
+        Append<LanguageSavedWebhookEvent>();
+        return this;
+    }
```

Comment:
And then a method `WebhookAllTheThings `that calls all 3 at once ?? 😁😇

Reply:
haha will gladly add a `WebHookAllTheThings` method once I'm done 😝